index.js

```diff
@@ -428,9 +432,14 @@ const plugin = fp(async function (app, opts) {
       }
 
       try {
-        document = parse(source)
+        document = parse(source, gqlParseOpts)
       } catch (syntaxError) {
         try {
+          // Do not try to JSON.parse maxToken exceeded validation errors
+          if (gqlParseOpts.maxTokens && syntaxError.message === `Syntax Error: Document contains more that ${gqlParseOpts.maxTokens} tokens. Parsing aborted.`) {
```

Comment:
relying on an external library's error message to decide what to do doesn't feel very reliable

Reply:
Totally agreed. Sadly I could not find another option. Any suggestion?

I did protect against it's potential change in the unit tests below.

Reply:
I guess we don't have many options here either. What will happen is that when dependabot bumps this dependency, the unit test will break which will give us some clue of what needs fixing.